src/FormMigration/FormMetaDecorator.php

```diff
@@ -761,4 +763,27 @@ public function getGiftAidDeclarationForm(): string
     {
         return $this->getMeta('give_gift_aid_declaration_form');
     }
+
+    /**
+     * @unreleased
+     */
+    public function getFormFeaturedImage()
+    {
+        $templateSettings = $this->getFormTemplateSettings();
+
+        if ( ! empty($templateSettings['introduction']['image'])) {
+            // Sequoia Template (Multi-Step)
+            $featuredImage = $templateSettings['introduction']['image'];
+        } elseif ( ! empty($templateSettings['visual_appearance']['header_background_image'])) {
+            // Classic Template - it doesn't use the featured image from the WP default setting as a fallback
+            $featuredImage = $templateSettings['visual_appearance']['header_background_image'];
+        } elseif ( ! isset($templateSettings['visual_appearance']['header_background_image'])) {
+            // Legacy Template or Sequoia Template without the ['introduction']['image'] setting
+            $featuredImage = get_the_post_thumbnail_url($this->form->id, 'full');
+        } else {
+            $featuredImage = '';
```

Comment:
Nit: I recommend using `null` to represent a non-value instead of an empty string. An empty string is still a thing, while `null` is a special value that specifically indicates "nothing".

Reply:
Resolved in add96a3c1627653656abbc44c014bfea0d5e2639